### PR TITLE
Fix: Correct Display of Mathematical Equations in Documentation

### PR DESCRIPTION
**Pull request by jawadhussein462**
Fix issue #480

### CODE REVIEW

File: examples/multilabel_classification/1-quickstart/plot_tutorial_multilabel_classification.py [hunk 1]
```diff
@@ -102,16 +102,16 @@
 # Bernstein and Waudby-Smith–Ramdas).
 # The two methods give two different guarantees on the risk:
 #
-# * RCPS: :math:`P(R(\mathcal{T}_{\hat{\lambda}})\leq\alpha)\geq 1-\delta`
-#   where :math:`R(\mathcal{T}_{\hat{\lambda}})`
```
Comment (Valentin-Laurent) [inline examples/multilabel_classification/1-quickstart/plot_tutorial_multilabel_classification.py:106]: That one is broken. You can use this tool (an online reStructured editor) to translate : https://feat.dlup.link/rsted

File: examples/multilabel_classification/1-quickstart/plot_tutorial_multilabel_classification.py [hunk 2]
```diff
@@ -102,16 +102,16 @@
...
-#   is the risk we want to control and :math:`\alpha` is the desired risk
+# * RCPS: ``P(R(𝒯ₓ̂ₗ) ≤ α) ≥ 1 - δ``
+#   where ``R(𝒯ₓ̂ₗ)``
+#   is the risk we want to control and α is the desired risk
 #
-# * CRC: :math:`\mathbb{E}\left[L_{n+1}(\hat{\lambda})\right] \leq \alpha`
-#   where :math:`L_{n+1}(\hat{\lambda})` is the risk of a new observation and
```
Comment (Valentin-Laurent) [inline examples/multilabel_classification/1-quickstart/plot_tutorial_multilabel_classification.py:110]: That one is broken. You can use this tool (an online reStructured editor) to translate : https://feat.dlup.link/rsted

File: examples/multilabel_classification/1-quickstart/plot_tutorial_multilabel_classification.py [hunk 3]
```diff
@@ -223,20 +223,20 @@
...
 # The procedure consist in multiple hypothesis testing. This is why the output
-# of this procedure isn't reduce to one value of :math:`\lambda`.
+# of this procedure isn't reduce to one value of ``λ``.
 #
-# More precisely, we look after all the :math:`\lambda` that sastisfy the
+# More precisely, we look after all the ``λ`` that sastisfy the
 # following:
-# :math:`\mathbb{P}(R(\mathcal{T}_{\lambda}) \leq \alpha ) \geq 1 - \delta`,
-# where :math:`R(\mathcal{T}_{\lambda})` is the risk we want to control and
-# each :math:`\lambda`` should satisfy FWER control.
```
Comment (Valentin-Laurent) [inline examples/multilabel_classification/1-quickstart/plot_tutorial_multilabel_classification.py:232]: That one is broken. You can use this tool (an online reStructured editor) to translate : https://feat.dlup.link/rsted

**Review by Valentin-Laurent — COMMENTED** [review]
Great job, it looks even better than what I had in mind.
Few broken "translations"

File: examples/multilabel_classification/1-quickstart/plot_tutorial_multilabel_classification.py (hunk 3 above)
Comment (jawadhussein462) [reply examples/multilabel_classification/1-quickstart/plot_tutorial_multilabel_classification.py:232]: Done!

File: examples/multilabel_classification/1-quickstart/plot_tutorial_multilabel_classification.py (hunk 2 above)
Comment (jawadhussein462) [reply examples/multilabel_classification/1-quickstart/plot_tutorial_multilabel_classification.py:110]: Done!

File: examples/multilabel_classification/1-quickstart/plot_tutorial_multilabel_classification.py (hunk 1 above)
Comment (jawadhussein462) [reply examples/multilabel_classification/1-quickstart/plot_tutorial_multilabel_classification.py:106]: Done!

**Review by Valentin-Laurent — APPROVED** [review]

**Pull request merged**